examples/llm/pretrain/default_executor.py

```diff
@@ -21,11 +21,9 @@
 
 def local_executor_torchrun(devices: int = 2) -> run.LocalExecutor:
     env_vars = {
-        "TRANSFORMERS_OFFLINE": "1",
-        "TORCH_NCCL_AVOID_RECORD_STREAMS": "1",
-        "NCCL_NVLS_ENABLE": "0",
-        "NVTE_DP_AMAX_REDUCE_INTERVAL": "0",
-        "NVTE_ASYNC_AMAX_REDUCTION": "1",
```

Comment:
n00b question: why can we drop setting these now?

Reply:
We don't use these env vars in TE any more.